### ISSUE TITLE
Refactor command line argument parsing

### ISSUE THREAD

**Issue by georgemccabe** [dtcenter/METplus#510]
Currently the parsing is done inside config_metplus. The parsing of the command line should really be done inside the script that is called, master_metplus.py, then the list of values should be passed into config_metplus for processing.

## Describe the Enhancement ##
This enhancement would allow the pytests to be run correctly. They are currently called by passing arguments to pytest which are handled because they are valid arguments to pytest that are used in the wrong way. This has lead to issues running the tests. If a test requires command line arguments (conf files) to run, these should be set inside the test themselves by passing them directly into config_metplus. Once this is done, we can simply run 'pytest' and it will run all of the existing tests, removing the need for a control script. Also, pytest's code coverage functionality does not work with the current logic.

### Time Estimate ###
*1-3 days*

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Funding Source ###
*Define the source of funding and account keys here or state NONE.*

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [x] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones
- [x] Select **milestone**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/NCAR/METplus/issues/new/choose), [MET](https://github.com/NCAR/MET/issues/new/choose), [METdb](https://github.com/NCAR/METdb/issues/new/choose), [METviewer](https://github.com/NCAR/METviewer/issues/new/choose), [METexpress](https://github.com/NCAR/METexpress/issues/new/choose), [METcalcpy](https://github.com/NCAR/METcalcpy/issues/new/choose), [METplotpy](https://github.com/NCAR/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://ncar.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the issue definition above.
- [x] Fork this repository or create a branch of **develop**.
Branch name:  `feature_<Issue Number>_<Description>`
- [x] Complete the development and test your changes.
- [x] Add/update unit tests.
- [x] Add/update documentation.
- [x] Push local changes to GitHub.
- [x] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [x] Iterate until the reviewer(s) accept and merge your changes.
- [x] Delete your fork or branch.
- [x] Close this issue.

**Comment by georgemccabe** [comment]
This issue also needs to be completed for @CPKalb 's driver script work (issue #?).

These changes involve:
* Moved command line argument handling into master_metplus.py (currently in config_metplus)
* config_metplus.setup will instead take a list of configs (conf files or variable overrides) to use to setup the METplusConfig object
